Twitter.py

```diff
@@ -0,0 +1,27 @@
+import tweepy
+
+
+
+class Twitter(object):
+    @staticmethod
+    def home_timeline(account, count):
+        auth = tweepy.OAuthHandler(TWITTER_CONSUMER_KEY, TWITTER_CONSUMER_SECRET)
+        auth.set_access_token(TWITTER_ACCESS_TOKEN,
+                              TWITTER_ACCESS_TOKEN_SECRET)
```

Comment:
These parameters need to be set via the environment (`os.environ`), and their absence handled.